applications/webhooks/src/webhooks.app.src

```diff
@@ -9,6 +9,7 @@
                  , crypto
 
                  , kazoo_apps
+                 , kazoo_bindings
```

Comment:
@jamhed if  `kazoo_apps` depends on `kazoo_bindings` there's no need to add it here.